src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -127,9 +127,10 @@ private void doSeek(JsonObject bodyAsJson) {
             this.seek(topicPartition, offset, fut.completer());
         }
 
+        final RoutingContext rc = routingContext;
```

Comment:
it's a fair point but the `handle` method comes from the `BridgeEndpoint` interface which is in common with AMQP bridging where the routing context has no meaning.

Reply:
I'm not asking to change signature of the `handle()` method, just for it to get the RC and then pass it to the private methods in HttpSinkBridgeEndpoint, rather than using a field. Am I missing something obvious?

In any case, I think we've stretched the abstractions which try to paper over the differences between AMQP and HTTP beyond their useful limits. _Even if_ we keep support for AMQP, I suspect the protocol and API interactions are just too different to really be worth abstracting over.

Reply:
@tombentley no you are right, I was thinking about that just now and I didn't read your comment. I will make this change as well before merging.

Reply:
Thanks @ppatierno